src/current/_includes/metrics-usage/sql.txn.rollback.count.md

```diff
@@ -0,0 +1 @@
+This metric shows the number of orderly transaction <a href="https://www.cockroachlabs.com/docs/stable/rollback-transaction">transactions</a>. A persistently high number of rollbacks may negatively impact the workload performance and needs to be investigated.
```

Comment:
I think this is a typo: "orderly transaction transactions"

did you mean "orderly transaction rollbacks" ?

Reply:
corrected